en/docs/security-announcements/security-advisories/2026/WSO2-2026-5059.md

```
@@ -0,0 +1,70 @@
+| WSO2 API Manager         | 4.5.0             | 55             |
+| WSO2 API Manager         | 4.4.0             | 70             |
+| WSO2 API Manager         | 4.3.0             | 106            |
+| WSO2 API Manager         | 4.2.0             | 195            |
+| WSO2 Identity Server     | 7.2.0             | 12             |
+| WSO2 Identity Server     | 7.1.0             | 53             |
+| WSO2 Identity Server     | 7.0.0             | 144            |
+| WSO2 Identity Server     | 6.1.0             | 266            |
+| WSO2 Identity Server     | 6.0.0             | 263            |
+
+### CREDITS
+WSO2 thanks, **[Maneesha Dewmine]()** for responsibly reporting the identified issue and working with us as we addressed it.
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)